### PR TITLE
Glpi connector

### PR DESCRIPTION
**Pull request by ianlcz**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: "3.8"
```
Comment (aurelien-alet) [inline docker-compose.yml:1]: This file does not belong here, it was just used for tests.

File: .env-example [hunk 2]
```diff
@@ -32,6 +32,16 @@ OTTM_JIRA_TOKEN=
 # Issue type filters for jira issues, comma separated
 OTTM_JIRA_ISSUE_TYPE=
 OTTM_TARGET_DATABASE=sqlite:///data/${OTTM_SOURCE_PROJECT}.sqlite3
+# MariaDB Authentification
+MARIADB_ROOT_PASSWORD=diouxx
```
Comment (aurelien-alet) [inline .env-example:36]: MARIADB env variables have to be removed, they were just useful for glpi test instance

File: connectors/glpi.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+from utils.date import date_to_datetime
+from utils.timeit import timeit
+
+class GlpiConnector:
+
+    def __init__(self, project_id, session, config) -> None:
+        self.config = config
+        self.session = session
+        self.project_id = project_id
+        self.glpi = GLPI(url=self.config.glpi_base_url,
```
Comment (aurelien-alet) [inline connectors/glpi.py:17]: It would be better to have a private attribute here : self.__glpi

File: connectors/glpi.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                         apptoken=self.config.glpi_app_token,
+                         auth=(self.config.glpi_username, self.config.glpi_password))
```
Comment (aurelien-alet) [inline connectors/glpi.py:19]: Both app token and username / password are needed ?

File: connectors/glpi.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+        return last_issue_date
+
+    def _get_issues(self, updated_after: datetime=None):
+        
+        if updated_after:
+            glpi_issues = []
+
+            for issue in self.glpi.get_all_items('Ticket'):
+                if date_to_datetime(issue['date_mod']) >= updated_after:
```
Comment (aurelien-alet) [inline connectors/glpi.py:56]: strinctly greater than

File: connectors/glpi.py [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+                    glpi_issues.append(issue)
+        else:
+            glpi_issues = self.glpi.get_all_items('Ticket')
```
Comment (aurelien-alet) [inline connectors/glpi.py:59]: maybe we need to add a config OTTM_GLPI_CATEGORY to filter tickets by category, we have to discuss about that

File: utils/date.py [hunk 7]
```diff
@@ -4,4 +4,7 @@ def date_iso_8601_to_datetime(date_str: str):
     return datetime.strptime(date_str, '%Y-%m-%dT%H:%M:%S.%f%z')
 
 def datetime_to_date_hours_minuts(date_datetime: datetime):
-    return date_datetime.strftime("%Y-%m-%d %H:%M")
\ No newline at end of file
+    return date_datetime.strftime("%Y-%m-%d %H:%M")
+
+def date_to_datetime(date_str: str):
```
Comment (aurelien-alet) [inline utils/date.py:9]: date_hour_to_datetime would be better

File: connectors/glpi.py (hunk 4 above)
Comment (ianlcz) [reply connectors/glpi.py:19]: Yes, token and username/password are needed to connect to GLPI and retrieve session token.

**Pull request merged**